html ve js dosyaları güncellenmmiştir home.css eklenmiştir.

OpenSoftware-World opened the pull request.

OpenSoftware-World committed:
html ve js dosyaları güncellenmmiştir home.css eklenmiştir.

Ledenav commented:
Home sayfasına güncelleme verilerini ve parantez içersinde ismini yazmaya ne dersin ? Bu arada 1 yıl sonra Merhaba

OpenSoftware-World commented:
> Home sayfasına güncelleme verilerini ve parantez içersinde ismini yazmaya ne dersin ? Bu arada 1 yıl sonra Merhaba

tamamdır düzelteceğim Merhabalar. Beraber çalışmaya ne dersiniz mesela birbirimizin yazılımlarına veya projelerine katkıda bulunmaya Olurmu

OpenSoftware-World committed:
Update lvw.css

OpenSoftware-World committed:
Update home.css

OpenSoftware-World committed:
Update home.html

 This page was edited by LinuxUsersLinuxMint
Spaces have been removed and the text has been corrected.

OpenSoftware-World committed:
Update index.html

Spaces have been removed and the text has been corrected.

OpenSoftware-World committed:
Update lvw.css

OpenSoftware-World committed:
Update lvw.js

OpenSoftware-World commented:
> Home sayfasına güncelleme verilerini ve parantez içersinde ismini yazmaya ne dersin ? Bu arada 1 yıl sonra Merhaba

Tamamdır Herşeyi tamamladım güncelleme verilerini ve adımı parantez içerisinde yazdım dosyalar sadece düzenlenmiştir herhangi bir ekleme/yenilik yapılmamıştır çekme isteğimi kabul edebebilirsiniz/merge isteğimi.

Bu arada Beraber çalışmaya ne dersiniz mesela birbirimizin yazılımlarına veya projelerine katkıda bulunmaya Olurmu

Ledenav commented:
Sanırım yanlış bişey yaptın , kontrol etmeye üşeniyorum,  kontrol eder misin

OpenSoftware-World commented:
> Sanırım yanlış bişey yaptın , kontrol etmeye üşeniyorum, kontrol eder misin

Tamamdır En Kısa zamanda kontrol edip hatamı düzelteceğim Hata İçin Özür dilerim :(

OpenSoftware-World commented:
> Sanırım yanlış bişey yaptın , kontrol etmeye üşeniyorum, kontrol eder misin

Tamamdır Hatayı çözdüm sanırsam merge edebilirsiniz yeni gönderdiğim çekme isteğini Sorun için Özür dilerim.

OpenSoftware-World commented:
> Sanırım yanlış bişey yaptın , kontrol etmeye üşeniyorum, kontrol eder misin

Sorunu Çözememişim :( Sorunu Araştıracağım

Ledenav commented:
🤒 Sanırım sorun düzelmedi, bence hatayı çözmelisin, eğer gerçekten öğrenmek istiyorsan, profilini okudum,  Teknolojiden yararlanmaktan çekinme, sana yardımcı olabileceği görüşündeyim,  daha önce hiç denemedim Yapay Zekayı bu işler için, üşengeçler için iyi olabilir sanki 🥱

OpenSoftware-World commented:
> 🤒 Sanırım sorun düzelmedi, bence hatayı çözmelisin, eğer gerçekten öğrenmek istiyorsan, profilini okudum, Teknolojiden yararlanmaktan çekinme, sana yardımcı olabileceği görüşündeyim, daha önce hiç denemedim Yapay Zekayı bu işler için, üşengeçler için iyi olabilir sanki 🥱

Tamamdır Tavsiyeleriniz İçin saol hatayı çözeceğim çözene kadar pes etmeyeceğim. Bu arada discord kullanıyormusunuz arkadaş olabiliriz.

OpenSoftware-World commented:
> 🤒 Sanırım sorun düzelmedi, bence hatayı çözmelisin, eğer gerçekten öğrenmek istiyorsan, profilini okudum, Teknolojiden yararlanmaktan çekinme, sana yardımcı olabileceği görüşündeyim, daha önce hiç denemedim Yapay Zekayı bu işler için, üşengeçler için iyi olabilir sanki 🥱

yeni bir çekme isteği oluşturdum sorun çözülmedi ancak yarın çalışmama devam edebilmem için bunu merge edebilirmisin sorun için tekrardan özür dilerim :(

OpenSoftware-World commented:
> 🤒 Sanırım sorun düzelmedi, bence hatayı çözmelisin, eğer gerçekten öğrenmek istiyorsan, profilini okudum, Teknolojiden yararlanmaktan çekinme, sana yardımcı olabileceği görüşündeyim, daha önce hiç denemedim Yapay Zekayı bu işler için, üşengeçler için iyi olabilir sanki 🥱

Sorun çözümü için geçici bir çözüm olarak eski lvw.js yi yükledim müsait olunca sorunu incelemeye alacağım.

OpenSoftware-World commented:
> 🤒 Sanırım sorun düzelmedi, bence hatayı çözmelisin, eğer gerçekten öğrenmek istiyorsan, profilini okudum, Teknolojiden yararlanmaktan çekinme, sana yardımcı olabileceği görüşündeyim, daha önce hiç denemedim Yapay Zekayı bu işler için, üşengeçler için iyi olabilir sanki 🥱

Sorunu çözdüm eski dosyaları yükledim ancak test etmedim bunu merge edebilirmisin. PATCH-5
